### DISCUSSION TITLE
Is open metadata supports Azure Synapse , Teradata and Sql DW?

### DISCUSSION THREAD

**Discussion by LingamAnitha** [open-metadata/OpenMetadata#7964] [Q&A] [answered]
Is open metadata supports Azure Synapse , Teradata and Sql DW?

**Comment by ayush-shah** [comment, marked as answer]
Hello @LingamAnitha, you can take a look at the list of connectors from this [Doc](https://docs.open-metadata.org/connectors/database).
If the connectors are not available in this list, you can raise a feature request from [here](https://github.com/open-metadata/OpenMetadata/issues/new?assignees=&labels=&template=feature_request.md&title=).
